src/flightRecorder.cpp

```diff
@@ -180,6 +180,7 @@ class Lookup {
         }
 
         jvmtiEnv* jvmti = VM::jvmti();
+        JNIEnv* jni = VM::jni();
```

Comment:
`VM::jni()` is not an expensive call, but it's not free either - not a trivial getter like `VM::jvmti()`.
So, I'd create `JNIEnv*` once in `Lookup` constructor and cache it as a field.

Reply:
Thanks, cached `JNIEnv`.